src/Symfony/Component/DependencyInjection/Compiler/RegisterEnvVarProcessorsPass.php

```diff
@@ -26,7 +27,7 @@
  */
 class RegisterEnvVarProcessorsPass implements CompilerPassInterface
 {
-    private static $allowedTypes = array('array', 'bool', 'float', 'int', 'string');
+    private static $allowedTypes = array('array', 'boolean', 'float', 'integer', 'string');
```

Comment:
@ro0NL can you submit another PR with only these changes that must be applied on 3.4, then we'll move this for 4.1, OK?

Reply:
Missed this. Will have a look ASAP :+1: